src/main/java/school/hei/haapi/endpoint/rest/validator/CreatePaymentValidator.java

```diff
@@ -9,8 +12,16 @@
 public class CreatePaymentValidator implements Consumer<CreatePayment> {
   @Override
   public void accept(CreatePayment createPayment) {
+    LocalDateTime now = LocalDateTime.ofInstant(Instant.now(), ZoneId.of("UTC+3"));
     if (createPayment.getAmount() == null) {
       throw new BadRequestException("Amount is mandatory");
     }
+    if (createPayment.getCreationDatetime() == null) {
+      throw new BadRequestException("Creation datetime is mandatory");
+    }
+    if (createPayment.getCreationDatetime().isAfter(Instant.now())) {
```

Comment:
use your "now" variable instead of Instant.now

Reply:
checked